templates/mattermost/files/README.md

```diff
@@ -26,7 +26,9 @@ The following changes have been made relative from initializing a Mattermost pro
 
 * The `.platform.app.yaml`, `.platform/services.yaml`, and `.platform/routes.yaml` files have been added.  These provide Platform.sh-specific configuration and are present in all projects on Platform.sh.  You may customize them as you see fit.
 * Mattermost binaries are downloaded during the build hook in `setup.sh`. You can edit that file to download a newer version of the upstream on a new environment to update.
-* Environment variables are set in the `.environment` file, which override default settings created in Mattermost's `/app/config/config.json` file, which allow  allow Mattermost to connect to PostgreSQL and Elasticsearch.
+* Environment variables are set in the `.environment` file, which override default settings created in Mattermost's `/app/config/config.json` file, which allow  allow Mattermost to connect to PostgreSQL and Elasticsearch. You can modify this file to override additional variables in `/app/config/config.json` by matching its keys to an exported variable prefixed by `MM_`, and with each key level separated by an underscore. For example, `port` is defined in `config.json` at `ServiceSettings.ListenAddress`, and over-written in `.environment` to read Platform.sh's `PORT` environment variable with `export MM_SERVICESETTINGS_LISTENADDRESS=":$PORT"`.
```

Comment:
The first sentence is a bit clunky with two "which" clauses together.  Can that be tidied up a bit?